docs/metrics/customizing-the-sdk/Program.cs

```diff
@@ -16,7 +16,12 @@ public class Program
     public static void Main()
     {
         using var meterProvider = Sdk.CreateMeterProviderBuilder()
-            .ConfigureResource(res => res.AddService("example-service"))
+            .ConfigureResource(r => r.AddAttributes(new List<KeyValuePair<string, object>>
+                {
+                    new KeyValuePair<string, object>("static-attribute1", "v1"),
```

Comment:
It's probably better to use a meaningful key instead of generic names. Something like MachineId, ProcessId, Region etc.

Reply:
Agree, will steal something from opentelemetry.io example in a subsequent PR.